sima/imaging.py

```diff
@@ -102,27 +102,27 @@ def __init__(self, sequences, savedir, channel_names=None,
                 raise Exception('Cannot initialize dataset without sequences '
                                 'or a directory.')
 
-            def unpack(sequence):
-                """Parse a saved Sequence dictionary."""
-                return sequence.pop('__class__')._from_dict(
-                    sequence, savedir)
-
             with open(join(savedir, 'dataset.pkl'), 'rb') as f:
                 data = pickle.load(f)
-            self.sequences = [unpack(s) for s in data.pop('sequences')]
+            if 'sequences' in data:
+                raise ImportError('Old version')
```

Comment:
Let's be a bit more specific about which version would have triggered this error (i.e. 1.0.0-dev?). Then, in some later version we will know whether it is likely safe to remove this check.